edi_835_parser/loops/service.py

```diff
@@ -143,7 +143,7 @@ def build(
 					return service, segment, segments
 
 				else:
-					message = f'Identifier: {identifier} not handled in service loop.'
+					message = f'Identifier: {identifier} not handled in service loop.  {segment}'
```

Comment:
noith, does that have a useful str?

Reply:
It just helps us find the "weird" when seeing the failure.   It really helps test authoring IMHO.